core/domain/user_domain.py

```diff
@@ -197,3 +197,42 @@ def remove_collection_id(self, collection_id):
         ids.
         """
         self.collection_ids.remove(collection_id)
+
+
+class PlayLaterActivities(object):
+    """Domain object for the play later activities model."""
+
+    def __init__(self, user_id, exploration_ids,
+                 collection_ids):
+        self.id = user_id
+        self.exploration_ids = exploration_ids
+        self.collection_ids = collection_ids
+
+    def add_exploration_id(
+            self, exploration_id, position_to_be_inserted=None):
+        """Inserts the exploration id at the given position. If position is not
+        specified, the exploration id is added at the end."""
+        if position_to_be_inserted:
+            self.exploration_ids.insert(position_to_be_inserted, exploration_id)
+        else:
+            self.exploration_ids.append(exploration_id)
+
+    def remove_exploration_id(self, exploration_id):
```

Comment:
Here and elsewhere: document args in the usual way.

The docstring should make clear what happens if the exploration_id is not in the list.